runtime/src/ext_intr.S

```diff
@@ -0,0 +1,54 @@
+
+.section .init.text, "ax"
+.align 2
+_ext_intr_handler:
+    // Clear MPIE: ensures that interrupts are disabled again after mret
+    csrw mscratch, t0
+    li t0, 0x80
+    csrc mstatus, t0
+    csrr t0, mscratch
+
+    // Return back to event loop
+    mret
+
+// meivt must point at an address in DCCM
+.section .data
```

Comment:
Isn't .rodata also in DCCM at this point?

Reply:
No, .rodata is in ICCM because it needs to be write protected (and DCCM is never write-locked).